test/Microsoft.TemplateEngine.TestHelper/Microsoft.TemplateEngine.TestHelper.csproj

```diff
@@ -3,7 +3,7 @@
   <PropertyGroup>
     <TargetFrameworks>$(NetCurrent);$(NetFrameworkCurrent)</TargetFrameworks>
     <IsPackable>true</IsPackable>
-    <IsTestUtilityProject>true</IsTestUtilityProject>
+    <IsShipping>false</IsShipping>
```

Comment:
The removal of the `IsTestUtilityProject` property is the actual fix in this PR.